src/main/webapp/WEB-INF/tags/instructor/studentList/filterBox.tag

```diff
@@ -65,6 +65,7 @@
                 <div class="checkbox" style="display:none;">
                     <input id="show_email" type="checkbox" checked>
                     <label for="show_email"><strong>Show Emails</strong></label>
+                    <button id="copy_email_button" class="btn btn-default btn-xs">Copy Emails</button>
```

Comment:
The `id` should be delimited by `-` rather than `_`. [See here](https://oss-generic.github.io/process/codingStandards/CodingStandard-Html.html#naming-convention)